lib/SIL/TypeLowering.cpp

```diff
@@ -1034,17 +1027,6 @@ namespace {
       llvm_unreachable("cannot destroy an UnsafeValueBuffer!");
     }
   };
-  
-  /// A class that acts as a stand-in for improperly recursive types.
-  class RecursiveErrorTypeLowering : public AddressOnlyTypeLowering {
-  public:
-    RecursiveErrorTypeLowering(SILType type)
-      : AddressOnlyTypeLowering(type) {}
-    
-    bool isValid() const override {
```

Comment:
It is used in [lib/SILOptimizer/Mandatory/PredictableMemOpt.cpp#44](https://github.com/apple/swift/blob/master/lib/SILOptimizer/Mandatory/PredictableMemOpt.cpp#L44), but since it's alway true, we can remove that check now. Right?